language/de/install.php

```diff
@@ -215,6 +214,14 @@
 	//
 	'EMAIL_CONFIG'	=> 'E-Mail-Konfiguration',
 
+	// Package info
+	'PACKAGE_VERSION'				=> 'Paketversion installiert',
+	'UPDATE_INCOMPLETE'				=> 'Deine phpBB-Installation wurde nicht richtig aktualisiert.',
+	'UPDATE_INCOMPLETE_MORE'		=> 'Bitte lies die folgenden Information, um diesen Fehler zu beheben.',
+	'UPDATE_INCOMPLETE_EXPLAIN'		=> '<h1>Unvollständige Aktualisierung</h1>
+
+		<p>Wir haben bemerkt, dass die letzte Aktualisierung deiner phpBB-Installation nicht vollständig durchgeführt wurde. Besuche die <a href="%1$s" title="%1$s">Datenbank-Aktualisierung</a>, stell sicher, dass <em>Nur die Datenbank aktualisieren</em> ausgewählt ist und klicke auf <strong>Absenden</strong>. Nachdem du die Datenbank erfolgreich aktualisiert hast, vergiss nicht das "install"-Verzeichnis zu löschen.</p>',
```

Comment:
Statt "Besuche die Datenbank-Aktualisierung würde ich "Rufe die Datenbank-Aktualisierung auf" schreiben (das ist nur meine Meinung, kein wirklicher Fehler)

>stell sicher,

ist aber in meinen Augen falsch, das muss "stelle sicher" lauten.

>und klicke auf <strong>Absenden</strong>

da würde ich noch ein "dann" oder "danach" einfügen:
>und klicke dann auf <strong>Absenden</strong>

Inhaltlich: Wann kommt der Hinweis? Ist der "Install" Ordner da zwingend vorhanden? Ansonsten sollte vielleicht noch ein Hinweis rein, dass man den erst mal (wieder) reinkopieren muss bevor man ihn entfernt. Da es außerdem um ein Update geht würde ich davon ausgehen, dass der Ordner schon mal weg war, daher gehört in meinen Augen ein "wieder" in den letzten Satz hinein.